.github/ISSUE_TEMPLATE

```diff
@@ -1,38 +0,0 @@
-<!--
-Please answer these questions before submitting your issue. Thanks!
-For questions please use one of our forums: https://github.com/golang/go/wiki/Questions
--->
-
-### What version of Go are you using (`go version`)?
-
-<pre>
-$ go version
-
-</pre>
-
-### Does this issue reproduce with the latest release?
-
-
-
-### What operating system and processor architecture are you using (`go env`)?
-
-<details><summary><code>go env</code> Output</summary><br><pre>
-$ go env
-
-</pre></details>
-
-### What did you do?
-
-<!--
-If possible, provide a recipe for reproducing the error.
-A complete runnable program is good.
-A link on play.golang.org is best.
--->
-
-
-
-### What did you expect to see?
-
-
-
-### What did you see instead?
```

Comment:
These are indeed good questions. Perhaps we move then to the wiki or something?

Reply:
Opened #87 to track this discussion. Are you alright with merging this PR as-is and adding new templates later?

Reply:
yes! we can do that ;) Thank you!